spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/web/reactive/server/MetricsWebFilter.java

```diff
@@ -64,10 +69,19 @@ public MetricsWebFilter(MeterRegistry registry, WebFluxTagsProvider tagsProvider
 
 	public MetricsWebFilter(MeterRegistry registry, WebFluxTagsProvider tagsProvider,
```

Comment:
I think we should deprecate this constructor in favour of the new one added below.